docs/book/src/topics/gpu.md

```diff
@@ -0,0 +1,161 @@
+# GPU-enabled clusters
+
+## Overview
+
+With CAPZ you can create GPU-enabled Kubernetes clusters on Microsoft Azure.
+
+Before you begin, be aware that:
+
+- [Scheduling GPUs](https://kubernetes.io/docs/tasks/manage-gpus/scheduling-gpus/) is a Kubernetes beta feature
+- [NVIDIA GPUs](https://docs.microsoft.com/en-us/azure/virtual-machines/sizes-gpu) are supported on Azure NC-series, NV-series, and NVv3-series VMs
+
+To deploy a cluster with support for NVIDIA GPU nodes, use the [nvidia-gpu flavor template](https://raw.githubusercontent.com/kubernetes-sigs/cluster-api-provider-azure/master/templates/cluster-template-nvidia-gpu.yaml).
+
+## An example GPU cluster
+
+Let's create a CAPZ cluster with an NVIDIA node and run a GPU-powered vector calculation.
+
+### Generate an nvidia-gpu cluster template
+
+Use the `clusterctl config cluster` command to generate a manifest that defines your GPU-enabled
+workload cluster.
+
+Remember to use the `nvidia-gpu` flavor with N-series nodes.
+
+```bash
+AZURE_CONTROL_PLANE_MACHINE_TYPE=Standard_D2s_v3 \
+AZURE_NODE_MACHINE_TYPE=Standard_NC6s_v3 \
+AZURE_LOCATION=southcentralus \
+clusterctl config cluster azure-gpu \
+  --kubernetes-version=v1.19.3 \
+  --worker-machine-count=1 \
+  --flavor=nvidia-gpu > azure-gpu-cluster.yaml
+```
+
+If you're a developer using `tilt`, you can set an N-series node type for the `nvidia-gpu` flavor in
+`tilt-settings.json` to override any default:
```

Comment:
sorry I meant adding the nvidia-gpu override to the example in https://github.com/kubernetes-sigs/cluster-api-provider-azure/tree/master/templates/flavors#defining-variable-overrides, I think the book docs are mostly meant to be user facing